<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Appointment</string>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>751</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>118</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>110</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Test Person 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Test Person 2</string>
     </property>
    </item>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>459</y>
      <width>751</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>calendarWidget</sender>
   <signal>clicked(QDate)</signal>
   <receiver>dateEdit</receiver>
   <slot>setDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>466</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
